docs/kagi/settings/billing.md

```diff
@@ -12,7 +12,6 @@ These settings control your billing options and status with Kagi, there are two
 Your billing settings will show you your [Plan Type](../plans/plan-types.md) as well as:
 
 * Total searches this period
-* Searches used by AI
```

Comment:
I just thought of something. Does the Trial plan have an AI interaction limit that displays in the billing page? If so, this could be left in but with a note explaining that.

Reply:
Yes it does.